community/interest-groups.md

```diff
@@ -23,8 +23,8 @@ The aim of the Interest Group is to discuss the use of MEI for the encoding of m
 
 ## Metadata and Cataloging Interest Group
 
-**Administrative Co-Chairs: [Margrethe Bue](mailto:Margrethe.Bue@nb.no) and [Kristina Richts](mailto:kristina.richts@uni-paderborn.de)**.<br/>
-The aims of the Metadata and Cataloging Interest Group is to discuss the use of MEI for the encoding of musical metadata; to make recommendations regarding the development of the element; to promote the sharing of and collaboration on tools for projects concerned with MEI metadata such as cataloging projects. 'Cataloging' in this context may include a wide range of projects such as thematic catalogues ('Werkverzeichnisse'), library catalogs, and other descriptions of musical works or sources. The group is open to all members of the MEI community. Members enroll simply by subscribing to the group's mailing list: [https://lists.uni-paderborn.de/mailman/listinfo/mei-catalog-ig](https://lists.uni-paderborn.de/mailman/listinfo/mei-catalog-ig).
+**Administrative Co-Chairs: [Sophia Dörner](mailto:sophia.doerner@hu-berlin.de) and [Dennis Ried](mailto:dennis.ried@uni-paderborn.de)**.<br/>
+The aim of the Metadata and Cataloging Interest Group is to discuss the use of MEI for the encoding of musical metadata; to make recommendations regarding the development of metadata specific elements; to promote the sharing of and collaboration on tools for projects concerned with MEI metadata such as cataloging projects. 'Cataloging' in this context may include a wide range of projects such as thematic catalogs ('Werkverzeichnisse'), library catalogs, and other descriptions of musical works or sources. The group is open to all members of the MEI community. Members enroll simply by subscribing to the group's mailing list: [https://lists.uni-paderborn.de/mailman/listinfo/mei-catalog-ig](https://lists.uni-paderborn.de/mailman/listinfo/mei-catalog-ig).
```

Comment:
```suggestion
The aim of the Metadata and Cataloging Interest Group is to discuss the use of MEI for the encoding of musical metadata; to make recommendations regarding the development of metadata-specific elements; to promote the sharing of and collaboration on tools for projects concerned with MEI metadata such as cataloging projects. 'Cataloging' in this context may include a wide range of projects such as thematic catalogs ('Werkverzeichnisse'), library catalogs, and other descriptions of musical works or sources. The group is open to all members of the MEI community. Members enroll simply by subscribing to the group's mailing list: [https://lists.uni-paderborn.de/mailman/listinfo/mei-catalog-ig](https://lists.uni-paderborn.de/mailman/listinfo/mei-catalog-ig).
```

Reply:
@musicEnfanthen Seems like I can't commit your change. Could you please open a separate PR?

Reply:
@riedde Dou you mind to create another PR to update the typo? Sorry for spotting it only after the merge.